WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -813,12 +813,8 @@ private void updateTitle() {
     }
 
     private void updateTitle(PageType pageType) {
-        if (pageType == PageType.MY_SITE && mSelectedSite != null) {
-            ((MainToolbarFragment) mBottomNav.getActiveFragment()).setTitle(mSelectedSite.getName());
-        } else {
-            ((MainToolbarFragment) mBottomNav.getActiveFragment())
-                    .setTitle(mBottomNav.getTitleForPageType(pageType).toString());
-        }
+        ((MainToolbarFragment) mBottomNav.getActiveFragment())
+                .setTitle(mBottomNav.getTitleForPageType(pageType).toString());
```

Comment:
It's out of scope of this PR but this whole logic is strange (and wrong). We don't actually need to update the title anywhere else than in the Bottom nav component since this is a callback triggered by the bottom nav. What we do here is that the bottom nav tells us that a tab was selected and we react to it by asking bottom nav for the active fragment and setting its title from the bottom nav.